src/components/Login.jsx

```diff
@@ -32,7 +32,22 @@ function Login() {
   }
 
   const responseGoogle = (response) => {
-    const userObject = jwt_decode(response.credential);
+    console.log(response.credential)
+    const responseStr = {credential: response.credential};
+    try {
+      fetch('users/decode', {
+        method: 'POST',
+        headers: {'Content-Type': 'application/json'},
+        body: JSON.stringify(responseStr)
+      })
+      .then(response => {
+        if(response.status === 200) {
+          navigate('/homepage');
+        }

```

Comment:
What if the response is not 200? What do we want to do about that? Note that say, a 400 status would not get caught in your catch block below